index.html

```diff
@@ -2,295 +2,317 @@
-                <div class="header-details">
-                  <p class="client">CANOPY</p>
-                  <span class="separator-circle"></span>
-                  <p class="role">Back End Dev</p>
-                  <span class="separator-circle"></span>
-                  <p class="year">2015</p>
-                </div>
-              </div>
-              <p class="project-details">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
-              <ul class="tags">
-                <li class="tag">html</li>
-                <li class="tag">css</li>
-                <li class="tag">javaScript</li>
-              </ul>
-              <div class="action">
-                <button class="btn-default" type="button">See Project</button>
+            <div class="bottom-arrow">
+              <div>
+                <a href="#">
+                  <span class="bottom"></span>
+                </a>
               </div>
             </div>
-          </article>
-          <article class="card bg-white">
-            <figure>
-              <img src="image/yoga.png" alt="yoga" />
-            </figure>
-            <div class="project-info">
-              <div class="project-header">
-                <h2 class="project-title">Tonic</h2>
-                <div class="header-details">
-                  <p class="client">CANOPY</p>
-                  <span class="separator-circle"></span>
-                  <p class="role">Back End Dev</p>
-                  <span class="separator-circle"></span>
-                  <p class="year">2015</p>
-                </div>
-              </div>
-              <p class="project-details">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
-              <ul class="tags">
-                <li class="tag">html</li>
-                <li class="tag">css</li>
-                <li class="tag">javaScript</li>
-              </ul>
-              <div class="action">
-                <button class="btn-default" type="button">See Project</button>
-              </div>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <!-- Divide Arrow -->
+    <div class="bottom-arrow1">
+      <div>
+        <a href="#">
+          <span class="bottom1"></span>
+        </a>
+      </div>
+    </div>
+    <!-- Works Section -->
+    <section class="work-section">
+      <h2 class="works-title">My Recent Works</h2>
+      <div class="projects-container">
+        <div class="project">
+          <div class="project-image"></div>
+          <div class="project-description">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
```

Comment:
- [ ] kindly reduce the space between the technologies to match Figma design

![image](https://user-images.githubusercontent.com/89282221/187992717-3d7c00d2-1dd1-41af-81b2-2a336eb822f3.png)

Reply:
Thank you very much for the feedback.
In FIGMA design padding is mentioned as 8px 12px. I made the padding equal to that in my CSS file. I am attaching the screenshot after making that. 
![Screenshot (38)](https://user-images.githubusercontent.com/106825373/188014539-4731d8a1-937f-4eb9-8106-afdd2dabcad0.png).
I think it is ok now.